entry.go

```diff
@@ -169,7 +182,7 @@ func (te *TunaEntry) Start() {
 	<-te.closeChan
 }
 
-func (te *TunaEntry) StartReverse(stream *smux.Stream) error {
+func (te *TunaEntry) StartReverse(stream *smux.Stream, conn net.Conn) error {
```

Comment:
Why we need to add conn in parameter? It seems the only place it's used is to close it. But it's closed anyway when StartReverse is returned